src/data/services/LicenseManagerApiService.js

```diff
@@ -0,0 +1,13 @@
+import apiClient from '../apiClient';
+import { configuration } from '../../config';
+
+class LicenseManagerApiService {
+  static licenseManagerBaseUrl = `${configuration.LICENSE_MANAGER_BASE_URL}/api/v1/`;
```

Comment:
nit: remove the trailing `/` and add it below between `${licenseManagerBaseUrl}/subscriptions`.